### DISCUSSION TITLE
[Chapter 02] POJO vs. JavaBeans

### DISCUSSION THREAD

**Discussion by corock** [cowaine/spring-boot-for-msa#6] [Chapter 02. Framework and Spring Boot] [answered]
## Questions

1. 스프링을 보다 보면 맨날 POJO, POJO 거리는데... 면접 질문으로 "POJO 예시를 들어보세요." 라고 받으면 어떻게 답변할 건가요?
2. _p. 70_ 3번째 문단에 EJB 를 설명하면서 _JavaBeans_ 용어가 나옵니다. 그럼, 이 녀석은 POJO 랑 어떤 차이점이 있는지 궁금합니다.

**Comment by JoisFe** [comment, marked as answer]
[참고 자료](https://doozi316.github.io/java/2020/07/01/WEB21/)
[참고 자료 2](https://levelup.gitconnected.com/what-do-pojo-and-bean-mean-in-java-how-are-they-different-latest-9555824e13a9)
해당 블로그글이 정말 설명이 되어있어 공유드립니다.

두 블로그에서 말하는 공통된 1줄 요약이 
`
All java beans are POJOs but vice-versa is not true.
`

- 모든 Java Bean은 POJO이나 반대는 아니다
- POJO는 특정 기술에 종속되지 않은 순수한 JAVA Object를 의미하는데 그 중에 하나의 예시가 Java Bean 으로 보면 될 것 같습니다.
